Client/Assets/ContextMenu.js

```diff
@@ -1,175 +1,127 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+///* This Source Code Form is subject to the terms of the Mozilla Public
+// * License, v. 2.0. If a copy of the MPL was not distributed with this
+// * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 (function() {
 
-"use strict";
+const GESTURE_ALLOWABLE_MOVEMENT = 10;
+const GESTURE_MINIMUM_PRESS_DURATION = 500;
 
-var MAX_RADIUS = 9;
+const SUPPRESS_NEXT_CLICK_DURATION = 50;
 
+var target = null;
+var originalTargetTouchCalloutStyle = '';
+
+var didLongPress = false;
```

Comment:
Nit: Use `let`. If we're moving to `const` we should just drop `var` entirely.

Reply:
I think we intend on uplifting this to `v9.x`, so we can't use `let` there (support began with iOS 10) :-/

https://caniuse.com/#feat=let